pkg/kubebench/scanner.go

```diff
@@ -149,6 +149,7 @@ func (s *Scanner) prepareKubeBenchJob(node corev1.Node) (*batchv1.Job, error) {
 					RestartPolicy:                corev1.RestartPolicyNever,
 					HostPID:                      true,
 					NodeName:                     node.Name,
+					Affinity:                     starboard.DefaultAffinity(),
```

Comment:
For kube-bench we explicitly set the NodeName property. If we avoid creating a scan job for windows nodes the node affinity would be redundant.